esp-hal/src/twai/mod.rs

```diff
@@ -186,6 +243,17 @@ impl embedded_can::Error for ErrorKind {
     }
 }
 
+/// Specifies in which mode the TWAI controller will operate.
+pub enum TwaiMode {
+    /// Normal operating mode
+    Normal,
+    /// Self-test mode (no acknowledgement required for a successful message
+    /// transmission)
+    NoAck,
```

Comment:
```suggestion
    SelfTest,
```

Sorry this is really nit-picky, but I think we should follow the naming given in the TRM (unless there was some other reason you have renamed this variant).

Reply:
Agree on that